### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by fonminer** [djm34/ccminer-msvc2015#39]
Zoin Coin Support - Lyralyra2zoin GPU

Only CPU mining currently possible

**Comment by djm34** [comment]
wut ? link ?

**Comment by fonminer** [comment]
https://github.com/zoinofficial/zoin

http://zoinofficial.com/

http://explorer.zoinofficial.com/

**Comment by djm34** [comment]
as far as I can see from the code, it looks a lot like standard lyra2Z

**Comment by fonminer** [comment]
Yes but there must be some minor changes, no GPU miner for it exist yet

**Comment by djm34** [comment]
well I think they are using lyra2 with 330,256 (row, column) 
This is pretty much pointless to make gpu miner for that as it will be a lot slower than cpu.

Aside from that, this is just a clone from zcoin...

**Comment by fonminer** [comment]
thanks all the same
